lib/plugins/aws/lib/naming.js

```diff
@@ -174,7 +174,7 @@ module.exports = {
 
   // S3
   getDeploymentBucketLogicalId() {
-    return 'ServerlessDeploymentBucket';
```

Comment:
This and the test below are the only changes that are actually needed? Everything else if just references.